src/main/java/org/stellar/sdk/SorobanServer.java

```diff
@@ -122,8 +116,12 @@ public TransactionBuilderAccount getAccount(String accountId)
     if (entries == null || entries.isEmpty()) {
       throw new AccountNotFoundException(accountId);
     }
-    LedgerEntry.LedgerEntryData ledgerEntryData =
-        ledgerEntryDataFromXdrBase64(entries.get(0).getXdr());
+    LedgerEntry.LedgerEntryData ledgerEntryData;
+    try {
```

Comment:
```suggestion
    try {
    LedgerEntry.LedgerEnt ledgerEntryData = LedgerEntry.LedgerEntryData.fromXdrBase64(entries.get(0).getXdr());
    
     long sequence = ledgerEntryData.getAccount().getSeqNum().getSequenceNumber().getInt64();
    return new Account(accountId, sequence);
    
    catch () {...}
```

rather than declare `ledgerEntryData` outside of try/catch, just use it inside when it's valid.

Reply:
I personally prefer to include as little code as possible within the try block. Assuming there is some other business logic before returning Account, should we also place it within the try block?